src/scripts/unlockMineAccount.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Minor detail: I'm not sure if this script is necessary, maybe we should add it to the PR description?